### PR TITLE
[Feature] Documentation on adding debate type.

### PR DESCRIPTION
**Pull request by jamiebrown201**
## Description
Add simple steps for someone to add their own debate type. Words are hard. Let me know spelling mistakes, grammatical errors and if I haven't explained it well. May also need to go more in depth. Let me know.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,6 +22,16 @@ You can get to the skeleton admin page here:
...
 ![Alt text](./docs/admin_examle.png?raw=true 'Example of how the admin currently looks')
 
+## Adding a new debate type
+
+When you add a new debate type you will also need to add the corresponding logic and view so the app knows how to handle the specific debate type. Here is a quick setup guide to what you need to do.
+
+1. Firstly in the `modules` folder in the root add a javascript file with the name of your debate type all in lowercase. e.g `foragainst.js`.
+
+2. In this file all you need to add is a function called display which takes three params `req`, `res` and `data` and ends with `res.render(data.type, data);`. You can manipulate the data object however you like for your specific debate.
+
```
Comment (Sqyphen) [inline README.md]: Would be nice to have the example code ready to copy, e.g.

```
function display(req, res, data) {
	res.render(data.type, data);
}
module.exports = { display };
```

File: README.md [hunk 2]
```diff
@@ -22,6 +22,16 @@ You can get to the skeleton admin page here:
...
+3. Next you need to create the view. In `views/debates` add a handlebars with the name of your debate type e.g `foragainst.hbs`
+
```
Comment (Sqyphen) [inline README.md]: Could we have the example code here too please, or would it be clearer to reference a template file?

```
{{#> head title='Name of debate' subtitle='debate description' type='debateTypeName' }}
{{/head}}

<body class="debate-debateTypeName">
	{{> user/header username=user.usernameNice }}

	{{> components/heading/topper debateType=debateType title=title description=description }}

	{{> components/heading/status debateOpen=debateOpen votingOpen=votingOpen }}

	<main class="o-grid-container">
		<div class="o-grid-row">
			<!-- Main page content goes here -->
		</div>
	</main>
	{{/ifThisOrThat}}

	{{> footer type='debateTypeName'}}
```

File: README.md [hunk 3]
```diff
@@ -22,6 +22,16 @@ You can get to the skeleton admin page here:
...
 
+## Adding a new debate type
+
+When you add a new debate type you will also need to add the corresponding logic and view so the app knows how to handle the specific debate type. Here is a quick setup guide to what you need to do.
+
+1. Firstly in the `modules` folder in the root add a javascript file with the name of your debate type all in lowercase. e.g `foragainst.js`.
+
+2. In this file all you need to add is a function called display which takes three params `req`, `res` and `data` and ends with `res.render(data.type, data);`. You can manipulate the data object however you like for your specific debate.
+
+3. Next you need to create the view. In `views/debates` add a handlebars with the name of your debate type e.g `foragainst.hbs`
```
Comment (Lily2point0) [inline README.md]: Also need to add front-end JS for a specific type?

File: README.md (hunk 2 above)
Comment (Sqyphen) [reply README.md]: Perhaps it should reference a template file to save updating the README on any changes?

File: README.md (hunk 3 above)
Comment (Sqyphen) [reply README.md]: Oh, and a .sass file too

File: README.md (hunk 1 above)
Comment (jamiebrown201) [reply README.md]: Added

File: README.md (hunk 2 above)
Comment (jamiebrown201) [reply README.md]: Added a really basic one to the readme.

File: README.md (hunk 3 above)
Comment (jamiebrown201) [reply README.md]: Good idea added that.

**Pull request merged**